File: web/src/connectivity/ice-policy/defaults.ts
import { ICEEndpointPool } from './endpoints'

export const EXISTING_DEFAULT_STUN_SERVER = 'stun:stun.l.google.com:19302'

// Preserve the shipped endpoint without inventing geography or availability.
export function defaultICEEndpointPool(): ICEEndpointPool {
 return new ICEEndpointPool([{
  id: 'shipped-google-stun', url: EXISTING_DEFAULT_STUN_SERVER,
  region: '', failureDomain: '', provider: '', family: 'any',
  trust: 'reviewed', priority: 0, enabled: true,
 }])
}

File: connectivity/icepolicy/defaults.go
package icepolicy

const ExistingDefaultSTUNServer = "stun:stun.l.google.com:19302"

// DefaultPool preserves the repository's shipped endpoint. Geography and
// infrastructure diversity remain unknown; this is not a reachability claim.
func DefaultPool() ICEEndpointPool {
	pool, _ := NewICEEndpointPool([]Endpoint{{
		ID: "shipped-google-stun", URL: ExistingDefaultSTUNServer,
		Family: "any", Trust: "reviewed", Enabled: true,
	}})
	return pool
}
